### DISCUSSION TITLE
Nice dashboard tab

### DISCUSSION THREAD

**Discussion by jans23** [borgbase/vorta#856] [Ideas]
Currently Vorta appears to be very technical-focused piece of software. One reason is that all windows (actually all tabs in the only window) are about settings. What I'm used to from other backup software instead is a start screen which gives an overview of the state and offers performing regular tasks instead. Such dashboard tab could cover aspects such as:

- Profile name: default (heading for the subsequent information)
- Last successful backup: yesterday 12:34
- Next scheduled backup: today 13:45
- Buttons: backup now, verify backup, settings

I think such dashboard-like tab would improve Vorta's appearance and usability.

**Comment by m3nu** [comment]
Could be nice. There are already some stats on the first tab. How about extending it and moving e.g. the compression settings (that are less important) somewhere else?

The repository and SSH are quite important, so those should stay in a central place.

**Comment by jans23** [comment]
I would clearly separate between dashboard and settings. Settings are supposed to be configured only once and stay unmodified for relatively long period. In contrast, a dashboard should contain status information and major buttons for every day use.

Let me elaborate my suggestion:

### Backup to NAS [This is the profile]

- Last successful backup: yesterday 12:34
- Next scheduled backup: today 13:45
- Buttons: start backup, restore backup, configure

### Backup to USB [This is another profile]

- Last successful backup: 1.6.2019 12:34
- Next scheduled backup: 30.6.2019 13:45
- Buttons: start backup, restore backup, configure

**Comment by ThomasWaldmann** [comment]
Maybe rather use ISO8601-like date/time formatting.

**Comment by ghost** [comment]
A kind of appealing welcome screen would be great, yes! After all, this is the first impression new users get of Vorta. As a relevant and inspiring art I would like to link https://elementary.io/docs/human-interface-guidelines#welcome-screen here.

**Comment by m3nu** [comment]
Right. That would be nice. PRs are always welcome.

**Comment by ghost** [comment]
> I would clearly separate between dashboard and settings.

related to the separation of Dashboard and Settings, as mentioned above by @jans23 , I would like to point to the following design considerations https://elementary.io/docs/human-interface-guidelines#accessible-configuration

@m3nu Thanks, I still have to become competent enough in Python for that ;-) Maybe you can realize that as Vorta developers as well?

**Comment by samuel-w** [comment]
My plan is for the main window to be 2 tabs: Overview and Global Settings (current Misc tab)
The Overview tab would have all the profiles, with each profile formatted like this (just a mockup):
![mockup](https://user-images.githubusercontent.com/67625312/92217766-c43e2e00-ee5d-11ea-87b0-eaf3d586a59b.png)

It would have the add profile and restore profile buttons here as well. The configure button would pop open a window with the tabs for per-profile options (Sources, Schedule, etc).

**Comment by ghost** [comment]
@samuel-w Thanks for your inspiration to work on it!
Your design still looks very very technical to me. Unfortunately the screenshot does not show the whole window, right?
Could you imagine something more appealing to non-technical users in particular?

**Comment by samuel-w** [comment]
I understand what you are saying.
New idea:
Keep the UI as is, except add an Overview tab and rename the Misc tab to Global Settings. This is more similar to the first comment's idea
Overview tab is composed of profiles, each formatted like this:
![mockup](https://user-images.githubusercontent.com/67625312/92272508-4a369500-eeaf-11ea-9630-e66cb0367791.png)

Each separated by a line.

**Comment by ghost** [comment]
@samuel-w This is not yet written code, is it? I am curious to see how this works in real code. Could you imagine building a dashboard prototype?

**Comment by ghost** [comment]
By the way: Will the possible renaming of the misc tab be part of this issue or should it be reported separately?

**Comment by m3nu** [comment]
> By the way: Will the possible renaming of the misc tab be part of this issue or should it be reported separately?

Just renaming a tab is quick. This issue will take longer to plan out and implement.

**Comment by samuel-w** [comment]
Renaming the misc tab is discussed in #580
